terraform_state_mv.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+# Terraform resource name of the resource group block where you want to move your state to
+# e.g. terraform state mv module.api.azurerm_resource_group.rg azurerm_resource_group.rg
+DEST_RG=$1
+
+# Terraform resource name of the app insights block where you want to move your state to
+# e.g. terraform state mv module.api.azurerm_application_insights.appinsights azurerm_application_insights.appinsights
+DEST_APPINSIGHTS=${1-""}
```

Comment:
maybe 2 scripts? one for app insights one for the RG?
that way it's easier to make it optional, apart from that I assume you meant this to be $2?